iree/compiler/Dialect/VMLA/README.md

```diff
@@ -8,4 +8,149 @@ to 8bit integers of 0 or 1).
 
 ## Adding an Op
```

Comment:
How about adding some further context before diving straight into "how to add an op" like "seeing which ops already exist" and/or "*when* to add a new op"?

The goal of a simple runtime with more work done at compile time is great, so it would be nice to start with guidance on how to achieve that.

Reply:
will do in another change - this is more about getting the 'how to add an op' in

Reply:
That's fair :)